VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -196,7 +193,9 @@
                         android:layout_height="@dimen/course_unit_nav_bar_btn_height"
                         android:background="@color/white"
                         android:text="@string/assessment_previous"
-                        android:textColor="?attr/buttonTextColor" />
+                        android:textColor="?attr/buttonTextColor"
+                        android:gravity="center_vertical|start"
```

Comment:
OK, then perhaps we should have an “EButton” that inherits from ETextView, and provides the correct accessibility layer. It doesn’t look like we’re using the standard button style anywhere, so we’ll always have to override the predefined styles if use the existing widget.
> On 23-Sep-2015, at 12:15 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/activity_course_base.xml <https://github.com/edx/edx-app-android/pull/371#discussion_r40175945>:
> 
> > @@ -196,7 +193,9 @@
> >                          android:layout_height="@dimen/course_unit_nav_bar_btn_height"
> >                          android:background="@color/white"
> >                          android:text="@string/assessment_previous"
> > -                        android:textColor="?attr/buttonTextColor" />
> > +                        android:textColor="?attr/buttonTextColor"
> > +                        android:gravity="center_vertical|start"
> Not a good idea, because, we are supporting accessibility. A button should be a button class's object, so that accessibility pronounces it e.g. "Next Button" otherwise TalkBack would just say "Next", which isn't what we want.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/371/files#r40175945>.
>